pmlc/dialect/pxa/transforms/stencil_gemm.cc

```diff
@@ -279,24 +368,29 @@ class StencilGEMM : public StencilBase {
     auto bodyBuilder = op.getBodyBuilder();
 
     auto tileAttr = bodyBuilder.getI64ArrayAttr(tileSize);
+    auto numBatchesAttr = bodyBuilder.getI64IntegerAttr(1);
 
     SmallVector<Value, 8> mapOperands;
     GemmOperand c(opC, {perm.indexes[0], perm.indexes[1]}, mapOperands);
     GemmOperand a(opA, {perm.indexes[0], perm.indexes[2]}, mapOperands);
     GemmOperand b(opB, {perm.indexes[2], perm.indexes[1]}, mapOperands);
 
-    auto gemm =
-        bodyBuilder.create<pxa::PxaGemmOp>(op.getLoc(), c.memref.getType(),  //
-                                           c.memref, c.accessMap, c.tileMap, //
-                                           a.memref, a.accessMap, a.tileMap, //
-                                           b.memref, b.accessMap, b.tileMap, //
-                                           tileAttr, mapOperands);
+    auto gemm = bodyBuilder.create<pxa::PxaGemmOp>(
+        op.getLoc(), c.memref.getType(), //
+        c.memref, AffineMapAttr::get(c.accessMap),
+        AffineMapAttr::get(c.tileMap), //
+        a.memref, AffineMapAttr::get(a.accessMap),
+        AffineMapAttr::get(a.tileMap), //
+        b.memref, AffineMapAttr::get(b.accessMap),
+        AffineMapAttr::get(b.tileMap), //
+        tileAttr, numBatchesAttr, mapOperands);
 
     opC.result().replaceAllUsesWith(gemm);
+    opC.erase();
   }
 
 public:
-  StencilGEMM(AffineParallelOp op, unsigned numThreads,
+  StencilGEMM(AffineParallelOp op, unsigned numThreads, std::string strategy,
```

Comment:
Use `llvm::StringRef`

Reply:
Generally, we shouldn't ever pass strings by-value because that causes an implicit copy.